static/app/views/alerts/list/rules/activatedRuleRow.tsx

```diff
@@ -0,0 +1,456 @@
+              activationStatus={
+                isWaiting ? ActivationStatus.WAITING : ActivationStatus.MONITORING
+              }
+            />
+          </Tooltip>
+        </FlexCenter>
+        <MarginLeft>{renderAlertRuleStatus()}</MarginLeft>
+      </FlexCenter>
+      <FlexCenter>
+        <ProjectBadgeContainer>
+          <ProjectBadge
+            avatarSize={18}
+            project={projectsLoaded && project ? project : {slug}}
+          />
+        </ProjectBadgeContainer>
+      </FlexCenter>
+
+      <FlexCenter>
+        {teamActor ? (
+          <ActorAvatar actor={teamActor} size={24} />
+        ) : (
+          <AssigneeWrapper>
+            {!projectsLoaded && (
+              <LoadingIndicator
+                mini
+                style={{height: '24px', margin: 0, marginRight: 11}}
+              />
+            )}
+            {projectsLoaded && (
+              <DropdownAutoComplete
+                data-test-id="alert-row-assignee"
+                maxHeight={400}
+                onOpen={e => {
+                  e?.stopPropagation();
+                }}
+                items={dropdownTeams}
+                alignMenu="right"
+                onSelect={handleOwnerChange}
+                itemSize="small"
+                searchPlaceholder={t('Filter teams')}
+                disableLabelPadding
+                emptyHidesInput
+                disabled={!hasEditAccess}
+              >
+                {({getActorProps, isOpen}) => (
+                  <DropdownButton {...getActorProps({})}>
+                    {avatarElement}
+                    {hasEditAccess && (
+                      <StyledChevron direction={isOpen ? 'up' : 'down'} size="xs" />
+                    )}
+                  </DropdownButton>
+                )}
+              </DropdownAutoComplete>
+            )}
+          </AssigneeWrapper>
+        )}
+      </FlexCenter>
+      <ActionsColumn>
+        <Access access={['alerts:write']}>
+          {({hasAccess}) => (
+            <DropdownMenu
+              items={actions}
+              position="bottom-end"
+              triggerProps={{
+                'aria-label': t('Actions'),
+                size: 'xs',
+                icon: <IconEllipsis />,
+                showChevron: false,
+              }}
+              disabledKeys={hasAccess && canEdit ? [] : ['delete']}
+            />
+          )}
+        </Access>
+      </ActionsColumn>
+    </ErrorBoundary>
+  );
+}
+
+const FlexCenter = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+const IssueAlertStatusWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  gap: ${space(1)};
+  line-height: 2;
+`;
+
+const AlertNameWrapper = styled('div')<{isIssueAlert?: boolean}>`
+  ${p => p.theme.overflowEllipsis}
+  display: flex;
+  align-items: center;
+  gap: ${space(2)};
+  ${p => p.isIssueAlert && `padding: ${space(3)} ${space(2)}; line-height: 2.4;`}
+`;
```

Comment:
Might be nice to use `static/app/components/profiling/flex.tsx` and `FlexContainer` where you can here. I think  you can cover most of the flex cases you have defined with that component. By reusing the component, it would likely cut down pretty substantially on the css payload of this component.

Reply:
🤔 
hmmm since most of this is just copy/pasta from the sister rules.tsx, i'm less inclined to rewrite the entire component

I can leave a note to come back to this though

Reply:
sounds good - this pattern also already exists in the code so nbd.